src/views/Home.vue

```diff
@@ -12,7 +12,7 @@
 				<CollectivesIcon />
 			</template>
 			<template #action>
-				<NcButton :aria-label="t('collectives', 'Create new collective')" :type="buttonType" @click="newCollective">
+				<NcButton :aria-label="t('collectives', 'Create new collective')" :variant="buttonType" @click="newCollective">
```

Comment:
I think you renamed `buttonType` to `buttonVariant` below. Maybe that's fixed in a later commit though...

Reply:
Well spotted! Fixed now.